definitions/mock/ibc.mock.proto

```diff
@@ -1,19 +1,25 @@
 syntax = "proto3";
 package ibc.mock;
 
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
 import "ibc/core/client/v1/client.proto";
 
 message Header {
   ibc.core.client.v1.Height height = 1;
-  uint64 timestamp = 2; 
+  google.protobuf.Timestamp timestamp = 2;
```

Comment:
Why is it necessary for the time to be of type `Timestamp` here?

Reply:
there is no particular reason - except google protobuf types are standard.

Reply:
Understood. Though still using the `Timestamp` can be a bit limiting IMO when trying to whip up quick tests. It's not exactly the most convenient. I think the mock setup should be as barebones as possible, with minimal dependencies and a straightforward structure.